kotest-framework/kotest-framework-engine/src/commonMain/kotlin/io/kotest/engine/config/applyConfigFromAbstractProjectConfig.kt

```diff
@@ -82,7 +82,6 @@ internal fun applyConfigFromProjectConfig(config: AbstractProjectConfig, configu
       }
    }
 
-   @Suppress("DEPRECATION") // Remove when removing Listener
-   val exts = config.listeners() + listOf(projectListener) + config.extensions() + config.filters()
+   val exts = listOf(projectListener) + config.extensions()
```

Comment:
should we rename `exts` to `extensions`?

Reply:
I'll do that in a follow up pr